tests/SchemeSpec.hs

```diff
@@ -226,10 +227,10 @@ testSigNonMalleability = do
     cmdWithWrongNumSig <- mkCommandTest [kp, wrongKp] [signer] "(somePactFunction)"
     shouldBeProcFail (verifyCommand cmdWithWrongNumSig)
 
-
 testSigsRoundtrip :: Spec
 testSigsRoundtrip = runIO $ do
   apiReq' "tests/sign-scripts/unsigned-exec.yaml" True True
   apiReq' "tests/sign-scripts/unsigned-cont.yaml" True True
   signReq "tests/sign-scripts/sign-req.yaml"
-  addSigsReq "tests/sign-scripts/add-sigs.yaml" False
+
+  BS.readFile "tests/sign-scripts/add-sigs.yaml" >>= addSigsReq ["tests/sign-scripts/key.yaml"] False
```

Comment:
You should look at adding coverage here for new cases you've added like out-of-order sigs etc